Prace_domowe/Praca_domowa1/Grupa4/Marcin Wilk/PD1.ipynb

```diff
@@ -0,0 +1,4755 @@
+{
```

Comment:
<p>Tu to by się przydały xticki, żeby było wiadomo, że wrzesień </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>